rake-tasks/browsers.rb

```diff
@@ -49,6 +49,17 @@
     :browser_name => "chrome",
     :available => chrome?
   },
+  "blackberry" => {
+    :python => {
+      :ignore => "blackberry",
+      :dir => "blackberry",
+      :file_string => "blackberry",
+      :class => "BlackBerry",
+      :constructor_args => "device_pass='password', bb_tools_dir='/Applications/Momentics.app/host_10_3_0_2702/darwin/x86/usr/bin'"
```

Comment:
probably makes sense to add to the python wiki about how to get the blackberry tools / install them. Also noting that this means that only mac users can run the blackberry tests.

If there's an environment variable that can be set for the tool location, would be better to use that i think.